### PR TITLE
Fixes Broken links in 'Node Status' section of 'Nodes' page

### PR DESCRIPTION
**Pull request by ADITYADAS1999**
Fixes Broken links in 'Node Status' section of 'Nodes' page

Links from https://kubernetes.io/docs/concepts/architecture/nodes/#node-status section are broken.

List of broken links
https://kubernetes.io/docs/concepts/node/node-status/#addresses
https://kubernetes.io/docs/concepts/node/node-status/#condition
https://kubernetes.io/docs/concepts/node/node-status/#capacity
https://kubernetes.io/docs/concepts/node/node-status/#info
https://kubernetes.io/docs/concepts/node/node-status

Issue : #42628

### CODE REVIEW

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -163,18 +163,18 @@ that should run on the Node even if it is being drained of workload applications
 
 A Node's status contains the following information:
 
-* [Addresses](/docs/concepts/node/node-status/#addresses)
-* [Conditions](/docs/concepts/node/node-status/#condition)
-* [Capacity and Allocatable](/docs/concepts/node/node-status/#capacity)
-* [Info](/docs/concepts/node/node-status/#info)
+* [Addresses](https://kubernetes.io/docs/reference/node/node-status/#addresses)
```
Comment (tengqm) [inline content/en/docs/concepts/architecture/nodes.md:166]: Please use relative paths, like this:

```suggestion
* [Addresses](/docs/reference/node/node-status/#addresses)
```

File: content/en/docs/concepts/architecture/nodes.md [hunk 2]
```diff
@@ -163,18 +163,18 @@ that should run on the Node even if it is being drained of workload applications
...
+* [Info](/docs/reference/node/node-status/#info)
 
 You can use `kubectl` to view a Node's status and other details:
 
 ```shell
 kubectl describe node <insert-node-name-here>
 ```
 
-See [Node Status](https://kubernetes.io/docs/reference/node/node-status/) for more details
+See [Node Status](/docs/reference/node/node-status/) for more details
```
Comment (tengqm) [inline content/en/docs/concepts/architecture/nodes.md:177]:
```suggestion
See [Node Status](/docs/reference/node/node-status/) for more details.
```

**Review by tengqm — COMMENTED** [review]
/approve

File: content/en/docs/concepts/architecture/nodes.md [hunk 3]
```diff
@@ -164,18 +164,18 @@ that should run on the Node even if it is being drained of workload applications
...
+* [Info](/docs/reference/node/node-status/#info)
 
 You can use `kubectl` to view a Node's status and other details:
 
 ```shell
 kubectl describe node <insert-node-name-here>
 ```
 
-See [Node Status](/docs/concepts/node/node-status) for more details.
+
```
Comment (ydFu) [inline content/en/docs/concepts/architecture/nodes.md:178]:
```suggestion
See [Node Status](/docs/reference/node/node-status/) for more details.
```
Missing this line.

File: content/en/docs/concepts/architecture/nodes.md [hunk 4]
```diff
@@ -184,7 +184,7 @@ availability of each node, and to take action when failures are detected.
 
 For nodes there are two forms of heartbeats:
 
-* Updates to the [`.status`](/docs/concepts/node/node-status/) of a Node
+* Updates to the [`.status`](/docs/reference/node/node-status/) of a Node
```
Comment (ydFu) [inline content/en/docs/concepts/architecture/nodes.md:187]:
```suggestion
* Updates to the [`.status`](/docs/reference/node/node-status/) of a Node.
```
Missing comma.

File: content/en/docs/concepts/architecture/nodes.md (hunk 3 above)
Comment (ADITYADAS1999) [reply content/en/docs/concepts/architecture/nodes.md]: Thanks for review this one @ydFu

**Comment by ydFu** [comment]
LGTM,
Use rebase to squash/fixup dummy/unnecessary commits into only one commit,
Then I can help you label `lgtm`.

File: content/en/docs/concepts/architecture/nodes.md [hunk 5]
```diff
@@ -184,7 +184,7 @@ availability of each node, and to take action when failures are detected.
 
 For nodes there are two forms of heartbeats:
 
-* Updates to the [`.status`](/docs/concepts/node/node-status/) of a Node
+* Updates to the [`.status`](/docs/reference/node/node-status/) of a Node,
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/architecture/nodes.md:187]:
```suggestion
* Updates to the [`.status`](/docs/reference/node/node-status/) of a Node.
```

**Comment by ADITYADAS1999** [comment]
> /label tide/merge-method-squash

Hey @ydFu 
In which branch I rebase to squash all commits into only one commit, ?

**Comment by ydFu** [comment]
> > /label tide/merge-method-squash
> 
> 
> 
> Hey @ydFu 
> 
> In which branch I rebase to squash all commits into only one commit, ?

To the latest commit that has completed the review, thanks

**Comment by ADITYADAS1999** [comment]
> > > /label tide/merge-method-squash
> > 
> > 
> > Hey @ydFu
> > In which branch I rebase to squash all commits into only one commit, ?
> 
> To the latest commit that has completed the review, thanks

That Sounds good

**Review by sftim — COMMENTED** [review]
Thanks for the fixes

/lgtm

**Comment by sftim** [comment]
I'm happy for Tide to squash-merge this.

**Pull request merged**